app/services/medical_professionals/update_service.rb

```diff
@@ -0,0 +1,14 @@
+class MedicalProfessionals::UpdateService
+  class << self
+    def update_doctor_or_mhp(first_name, last_name)
+      doctor = Doctor.find_by(first_name: first_name, last_name: last_name)
+      mhp = MentalHealthProfessional.find_by(first_name: first_name, last_name: last_name)
```

Comment:
Why not just find by id?

Reply:
> Why not just find by id?

I was thinking the same thing, however, at scale, there may be the same number of records in both tables. So both doctor and mhp records would be found as they don't have a unique identifier.